src/Models/ShallowWaterModels/shallow_water_diffusion_operators.jl

```diff
@@ -17,7 +17,7 @@ import Oceananigans.TurbulenceClosures:
                         calculate_nonlinear_viscosity!, 
                         viscosity,
                         with_tracers,
-                        calc_νᶜᶜᶜ,
+                        calc_nonlinear_νᶜᶜᶜ,
```

Comment:
Why this change?

Reply:
The rationale was to differentiate from `calc_νᶜᶜᶜ()` from `νᶜᶜᶜ()`, which at first seem to do the same thing. That said, I'm okay with other names. What would you suggest?

Reply:
Do we even need separate `calc_νᶜᶜᶜ()` and `νᶜᶜᶜ()` if they do the same thing?